cli/src/maintenance.rs

```diff
@@ -1648,4 +1704,36 @@ mod test {
             state.clock.slot = next_slot;
         }
     }
+
+    #[test]
+    fn test_below_epoch_threshold() {
+        let mut state = new_empty_solido();
+        state.stake_unstake_any_time = false;
+        state.clock.slot = 9194;
+        state.epoch_schedule.slots_per_epoch = 100;
+        state.clock.epoch = 91;
+        assert_eq!(state.should_perform_maintenance_at_end_of_epoch(), Some(()));
+    }
+
+    #[test]
+    fn test_above_equal_epoch_threshold() {
+        let mut state = new_empty_solido();
+        state.stake_unstake_any_time = false;
+        state.clock.slot = 9195;
+        state.epoch_schedule.slots_per_epoch = 100;
+        state.clock.epoch = 91;
+        assert_eq!(state.should_perform_maintenance_at_end_of_epoch(), None);
+        state.clock.slot = 9199;
+        assert_eq!(state.should_perform_maintenance_at_end_of_epoch(), None);
```

Comment:
Huh, aren’t these two tests the wrong way around? In `test_below_epoch_threshold` it should return `None` and here both should return `Some(())`? The implementation of `state.should_perform_maintenance_at_end_of_epoch` looks correct to me though. Am I missing something?

Reply:
Yeah, depends on the interpretation. Could be above the `1 - threshold`, but it's correct to swap the names 👍